src/ServiceControl/MessageFailures/Api/FailedMessageViewIndex.cs

```diff
@@ -30,6 +31,7 @@ select new
                     message.Status,
                     TimeSent = (DateTime) processingAttemptsLast.MessageMetadata["TimeSent"],
                     ReceivingEndpointName = ((EndpointDetails) processingAttemptsLast.MessageMetadata["ReceivingEndpoint"]).Name,
+                    QueueAddress = processingAttemptsLast.FailureDetails.AddressOfFailingEndpoint.ToLowerInvariant(),
```

Comment:
Will we need it when we filter by QueryAddress?

https://github.com/Particular/ServiceControl/pull/762/files#diff-59f9f660f25fbc5f2a406e858443f133R233